ui/push-health/ClassificationGroup.jsx

```diff
@@ -76,8 +74,13 @@ class ClassificationGroup extends React.PureComponent {
 
     return (
       <Row className={`justify-content-between ${className}`}>
-        <h4 className="w-100" onClick={this.toggleDetails}>
-          <span className={`pointable badge badge-${headerColor} w-100`}>
+        <h4 className="w-100">
+          <span
+            className={`pointable badge badge-${headerColor} w-100`}
+            onClick={this.toggleDetails}
```

Comment:
All the other changes seem innocous.
This move of the `onClick` from the `h4` to the `span` would need a quick testing.